06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,91 @@
+
+/**
+ * InvertedIndex class with constructor
+ * @class
+ */
+class InvertedIndex {
+
+  /**
+   * Creates an instance of InvertedIndex.
+   * @memberOf InvertedIndex
+   */
+  constructor() {
+    this.indexedFiles = {};
+    this.searchResults = {};
+  }
+
+  /** creates index and update the indexed files
+   * @param {object} uploadedFiles - object containing uploaded books
+   * @param {Sting} file - file name
+   * @return  {object}  - this.indexedFiles
+   */
+  updateIndexedFilesRecords(uploadedFiles, file) {
```

Comment:
I'm not sure I even know what this function is doing. The parameters alone `uploadedFiles, file` are confusing and the JSDoc does not do this function any justice.